forest/bonsai/tests/test_simulate_gps_data.py

```diff
@@ -1,12 +1,14 @@
 """Tests for simulate_gps_data module"""
 
+import datetime
+
```

Comment:
Extra blank line

Reply:
Isn't this right? Since datetime is a standard library.

Reply:
Yes, indeed!